.github/workflows/test.yaml

```diff
@@ -0,0 +1,19 @@
+name: Test
+on: push
+
+jobs:
+    js-test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup Node.js
+              uses: actions/setup-node@v2.5.1
+              with:
+                  node-version: "16.x"
```

Comment:
(I’d prefer `"16"` so it’s the same string as in `.nvmrc` but that’s a very minor nitpick)

Reply:
but that makes no difference in terms of which version will be chosen, right?

Reply:
done

Reply:
Yeah, should be purely cosmetic.